src/map/script.cpp

```diff
@@ -4235,7 +4235,7 @@ void script_attach_state(struct script_state* st){
 		sd->st = st;
 		sd->npc_id = st->oid;
 		sd->npc_item_flag = st->npc_item_flag; // load default.
-		sd->state.disable_atcommand_on_npc = (!pc_has_permission(sd, PC_PERM_ENABLE_COMMAND));
+		sd->state.disable_atcommand_on_npc = battle_config.atcommand_disable_npc;
```

Comment:
Probably better to put this into an "and" relation?
For example a server owner could want to disable it for all players by default, but wants to use it himself or allow it for GMs.
Like this:
```
sd->state.disable_atcommand_on_npc = battle_config.atcommand_disable_npc && (!pc_has_permission(sd, PC_PERM_ENABLE_COMMAND));
```
This would require to change the default option to "false" in groups.conf as well though.